frontend/src/citizen-frontend/header/Header.tsx

```diff
@@ -35,30 +35,41 @@ export default React.memo(function Header(props: { ariaHidden: boolean }) {
 
   const {
     unreadPedagogicalDocumentsCount,
-    refreshUnreadPedagogicalDocumentsCount
-  } = useContext<PedagogicalDocumentsState>(PedagogicalDocumentsContext)
+    unreadVasuDocumentsCount,
+    refreshUnreadPedagogicalDocumentsCount,
+    refreshUnreadVasuDocumentsCount
+  } = useContext<ChildDocumentsState>(ChildDocumentsContext)
 
   useEffect(() => {
-    if (user) refreshUnreadPedagogicalDocumentsCount()
-  }, [refreshUnreadPedagogicalDocumentsCount, user])
+    if (user) {
+      refreshUnreadPedagogicalDocumentsCount()
+      refreshUnreadVasuDocumentsCount()
+    }
+  }, [refreshUnreadVasuDocumentsCount, refreshUnreadPedagogicalDocumentsCount, user])
 
   const location = useLocation()
   const hideLoginButton = location.pathname === '/login'
 
+  const unreadDocumentCount = useCallback(() => {
+    return (
+      (unreadPedagogicalDocumentsCount || 0) + (unreadVasuDocumentsCount || 0)
+    )
+  }, [unreadPedagogicalDocumentsCount, unreadVasuDocumentsCount])
```

Comment:
Wrapping this in a `useCallback` hook and then just evaluating it immediately is just a waste of resources and it hurts readability. Just do 
```
const unreadDocumentCount = (unreadPedagogicalDocumentsCount ?? 0) + (unreadVasuDocumentsCount ?? 0)
```